### PR TITLE
chore: add api_shortname and library_type to repo metadata

### PR DESCRIPTION
**Pull request by parthea**
Update .repo-metadata.json as required by go/library-data-integrity

### CODE REVIEW

File: .repo-metadata.json [hunk 1]
```diff
@@ -11,5 +11,7 @@
   "codeowner_team": "@googleapis/ml-apis",
   "name_pretty": "Google Cloud Video Intelligence",
   "client_documentation": "https://cloud.google.com/nodejs/docs/reference/video-intelligence/latest",
-  "release_level": "ga"
+  "release_level": "stable",
+  "api_shortname": "video",
```
Comment (bcoe) [inline .repo-metadata.json:15]:
```suggestion
  "api_shortname": "videointelligence",
```

**Review by bcoe — APPROVED** [review]

**Comment by parthea** [comment]
Merge https://github.com/googleapis/nodejs-video-intelligence/pull/658 first

**Comment by tmatsuo** [comment]
I think we can close #658

**Comment by tmatsuo** [comment]
Closed #658

**Review by bcoe — APPROVED** [review]

**Pull request merged**